### ISSUE TITLE
Invalid credentials

### ISSUE THREAD

**Issue by nandodelauni** [mrmans0n/localio#16]
I can successfully load the translation using OAuth, however a few minutes later I try and it says invalid credentials:

```
Logging in to Google Drive...
Logged in!
Spreadsheet found. Analyzing...
Languages detected: en, es -- using en as default.
Building terminology in memory...
Loaded!
Writing iOS translations...
 > en
 > es
 > LocalizableConstants.h
Done!

Logging in to Google Drive...
Logged in!
Invalid Credentials
```

**Comment by mrmans0n** [comment]
What are you putting in the :access_token param? Make sure you are actually using the one given in the output of the first valid execution, and not the code that you had to paste the first time trying.

**Comment by nandodelauni** [comment]
Yes, I am using the one returned in the CLI right after saying this:

`\n**IMPORTANT** You can store your access_token in the Locfile :source parameter for avoiding this step in the future:\n`

**Comment by mrmans0n** [comment]
I am leaving this here for my own future reference: http://stackoverflow.com/questions/26789804/ruby-google-drive-gem-oauth2-saving

I still have to code the refreshing token thing. Probably this weekend.

**Comment by nandodelauni** [comment]
Awesome, thanks!

**Comment by mrmans0n** [comment]
I have just published an update, the 0.1.2 version. It won't need the `:access_token` anymore in the Locfile, so you can remove that (will be ignored anyway). 

However, the tokens (access and refresh) will be stored in a hidden file in the Locfile directory, and they should be able to update themselves there so only the first run config would be needed in the future.

Adding the config file `.localio.yml` to the .gitignore file of your project is recommended. Everything is detailed in the README.

Thanks for noticing!
